modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js

```diff
@@ -28,7 +28,10 @@ const formatActions = (actions, itemData) => {
 	return actions
 		? actions.reduce((actions, action) => {
 				if (action.data?.permissionKey) {
-					if (itemData.actions[action.data.permissionKey]) {
+					if (
+						itemData.actions &&
+						itemData.actions[action.data.permissionKey]
+					) {
```

Comment:
[optional] we can use optional chaining here, `itemData.actions?.[action.data.permissionKey]`